data/processed/hesa/total_postgraduates.csv

```diff
@@ -0,0 +1,201 @@
+99,2018/19,UKI3,2018,12575.0
+100,2014/15,UKJ4,2018,1135.0
+101,2015/16,UKJ4,2018,1170.0
+102,2016/17,UKJ4,2018,1195.0
+103,2017/18,UKJ4,2018,1175.0
+104,2018/19,UKJ4,2018,1180.0
+105,2014/15,UKD4,2018,1345.0
+106,2015/16,UKD4,2018,1240.0
+107,2016/17,UKD4,2018,1195.0
+108,2017/18,UKD4,2018,1295.0
+109,2018/19,UKD4,2018,1265.0
+110,2014/15,UKF2,2018,2235.0
+111,2015/16,UKF2,2018,2315.0
+112,2016/17,UKF2,2018,2200.0
+113,2017/18,UKF2,2018,2105.0
+114,2018/19,UKF2,2018,2250.0
+115,2014/15,UKF3,2018,240.0
+116,2015/16,UKF3,2018,255.0
+117,2016/17,UKF3,2018,255.0
+118,2017/18,UKF3,2018,260.0
+119,2018/19,UKF3,2018,275.0
+120,2014/15,UKD7,2018,1870.0
+121,2015/16,UKD7,2018,2065.0
+122,2016/17,UKD7,2018,2220.0
+123,2017/18,UKD7,2018,2210.0
+124,2018/19,UKD7,2018,2255.0
+125,2014/15,UKM5,2018,1015.0
+126,2015/16,UKM5,2018,1005.0
+127,2016/17,UKM5,2018,1010.0
+128,2017/18,UKM5,2018,1055.0
+129,2018/19,UKM5,2018,1015.0
+130,2014/15,UKE2,2018,1275.0
+131,2015/16,UKE2,2018,1270.0
+132,2016/17,UKE2,2018,1270.0
+133,2017/18,UKE2,2018,1255.0
+134,2018/19,UKE2,2018,1280.0
+135,2014/15,UKN0,2018,1665.0
+136,2015/16,UKN0,2018,1725.0
+137,2016/17,UKN0,2018,1755.0
+138,2017/18,UKN0,2018,1850.0
+139,2018/19,UKN0,2018,1760.0
+140,2014/15,UKC2,2018,2130.0
+141,2015/16,UKC2,2018,2140.0
+142,2016/17,UKC2,2018,2255.0
+143,2017/18,UKC2,2018,2245.0
+144,2018/19,UKC2,2018,2215.0
+145,2014/15,UKI5,2018,315.0
+146,2015/16,UKI5,2018,325.0
+147,2016/17,UKI5,2018,295.0
+148,2017/18,UKI5,2018,260.0
+149,2018/19,UKI5,2018,265.0
+150,2014/15,UKI6,2018,235.0
+151,2015/16,UKI6,2018,285.0
+152,2016/17,UKI6,2018,275.0
+153,2017/18,UKI6,2018,245.0
+154,2018/19,UKI6,2018,225.0
+155,2014/15,UKI7,2018,1120.0
+156,2015/16,UKI7,2018,980.0
+157,2016/17,UKI7,2018,870.0
+158,2017/18,UKI7,2018,825.0
+159,2018/19,UKI7,2018,890.0
+160,2014/15,UKG2,2018,450.0
+161,2015/16,UKG2,2018,410.0
+162,2016/17,UKG2,2018,410.0
+163,2017/18,UKG2,2018,365.0
+164,2018/19,UKG2,2018,370.0
+165,2014/15,UKE3,2018,2560.0
+166,2015/16,UKE3,2018,2560.0
+167,2016/17,UKE3,2018,2480.0
+168,2017/18,UKE3,2018,2435.0
+169,2018/19,UKE3,2018,2480.0
+170,2014/15,UKJ2,2018,2465.0
+171,2015/16,UKJ2,2018,2485.0
+172,2016/17,UKJ2,2018,2410.0
+173,2017/18,UKJ2,2018,2295.0
+174,2018/19,UKJ2,2018,2335.0
+175,2014/15,UKC1,2018,1260.0
+176,2015/16,UKC1,2018,1255.0
+177,2016/17,UKC1,2018,1275.0
+178,2017/18,UKC1,2018,1265.0
+179,2018/19,UKC1,2018,1285.0
+180,2014/15,UKM8,2018,3665.0
+181,2015/16,UKM8,2018,3770.0
+182,2016/17,UKM8,2018,3875.0
+183,2017/18,UKM8,2018,3975.0
+184,2018/19,UKM8,2018,4115.0
+185,2014/15,UKG3,2018,4435.0
+186,2015/16,UKG3,2018,4510.0
+187,2016/17,UKG3,2018,4545.0
+188,2017/18,UKG3,2018,4615.0
+189,2018/19,UKG3,2018,4790.0
+190,2014/15,UKL1,2018,1455.0
+191,2015/16,UKL1,2018,1490.0
+192,2016/17,UKL1,2018,1630.0
+193,2017/18,UKL1,2018,1625.0
+194,2018/19,UKL1,2018,1760.0
+195,2014/15,UKE4,2018,3110.0
+196,2015/16,UKE4,2018,3210.0
+197,2016/17,UKE4,2018,3210.0
+198,2017/18,UKE4,2018,3305.0
```

Comment:
In the schema the value is marked as an `int` but values are floats

Reply:
All integer types are now outputted as integer.